src/main/play-doc/general/MigrationGuide.md

```diff
@@ -1,18 +1,31 @@
 # ConductR Migration Guide
 
 
-This document describes what is required to move between version 1.1 to 1.2. Here is a list of major areas to be considered when migrating:
+This document describes what is required to move between version 1.1 to 2.0.
 
-* [Proxying](#Roles_and_toplogy)
+Here is a list of major areas to be considered when migrating:
+
+* [Binary incompatibility](#Binary_incompatibility)
+* [Topology change](#Topology_change)
 * [Proxying](#Proxying)
 
-## Roles and topology
 
-ConductR now requires roles to be specified for each node where ConductR bundles run. Prior to installing ConductR consider a topology that has roles representing a DMZ, gateway services, persistance services (databases) and so forth. You can disable role checking by specifying the following in `application.ini`: 
```

Comment:
@huntc and @markusjura - removed Roles from migration guide as it's still optional by default